src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeValidator.java

```diff
@@ -1,47 +1,67 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.validation;
 
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Sets;
 import org.apache.commons.collections4.MapUtils;
 import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.FileNameIrregularitiesException;
 import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.OcrDataNotFoundException;
 import uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType;
 import uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputEnvelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputScannableItem;
 import uk.gov.hmcts.reform.bulkscanprocessor.model.common.Classification;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.document.output.Pdf;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static java.util.stream.Collectors.toList;
+
 public final class EnvelopeValidator {
 
+    public static final Map<String, InputDocumentType> ocrDocumentTypePerJurisdiction =
+        ImmutableMap.of(
+            "SSCS", InputDocumentType.SSCS1
+        );
+
     private EnvelopeValidator() {
         // util class
     }
 
     /**
-     * Assert scannable item contains ocr data
+     * Assert scannable items contain ocr data
      * when envelope classification is NEW_APPLICATION
-     * and jurisdiction is SSCS
-     * and document type of the scannable item is SSCS1.
      * Throws exception otherwise.
      *
      * @param envelope to assert against
      */
     public static void assertEnvelopeContainsOcrDataIfRequired(InputEnvelope envelope) {
 
-        if (envelope.jurisdiction.equalsIgnoreCase("SSCS")
-            && Classification.NEW_APPLICATION.equals(envelope.classification)) {
+        if (Classification.NEW_APPLICATION.equals(envelope.classification)) {
 
-            boolean ocrDataExists = envelope.scannableItems
+            InputDocumentType typeThatShouldHaveOcrData = ocrDocumentTypePerJurisdiction.get(envelope.jurisdiction);
```

Comment:
This will cause us to fail when NEW_APPLICATION envelopes start showing up for other services. It would be better to reject them.

Reply:
apparently `NEW_APPLICATION` only makes sense if there is OCR present on a service specific document type.